core/src/main/scala/org/http4s/EntityEncoder.scala

```diff
@@ -137,13 +137,14 @@ trait EntityEncoderInstances extends EntityEncoderInstances0 {
     simple(hdr)(s => ByteVector.view(s.getBytes(charset.nioCharset)))
   }
 
-  implicit def charSequenceEncoder[A <: CharSequence](implicit charset: Charset = DefaultCharset): EntityEncoder[CharSequence] =
+  implicit def charBufferEncoder(implicit charset: Charset = DefaultCharset): EntityEncoder[CharBuffer] =
     stringEncoder.contramap(_.toString)
 
   implicit def charArrayEncoder(implicit charset: Charset = DefaultCharset): EntityEncoder[Array[Char]] =
-    charSequenceEncoder.contramap(new String(_))
+    stringEncoder.contramap(new String(_))
 
-  implicit val charEncoder: EntityEncoder[Char] = charSequenceEncoder.contramap(Character.toString)
+  implicit val charEncoder: EntityEncoder[Char] =
```

Comment:
Probably not a lot.  I did once implement an endpoint that returned '0' or '1', though I'll bet I didn't do it with a Char.